quality-control/human-clinical-and-health-data.md

```diff
@@ -1,27 +1,79 @@
+
+* Data linkage: Ensuring data quality is again key in this step, especially paying attention to data quality dimensions such as completeness, accuracy, timeliness or consistency among the different datasets to be linked. One approach is to use standards, specifically on the semantic and syntactic levels, as they facilitate the linkage of databases, especially if they come from different sources.
+
+  Continuing with the example above, the ISIN register is used in the Czech Republic to obtain epidemiological data on the occurrence of patients with infectious diseases (including patients with COVID-19).  It is a universal system whose purpose is to obtain information on the occurrence of infectious diseases to assess the development of the epidemiological situation in the Czech Republic, to monitor the health status of the population and to manage the provision of health care. Data from regional hygiene stations, laboratories and hospitals caring for hospitalised patients were centralised in the ISIN during the SARS-CoV-2 pandemic. The ISIN system is managed by The Institute of Health Information and Statistics of the Czech Republic, an organisational unit of the state and has been established by the Ministry of Health of the Czech Republic.
+
+* Sample related data: Each sample should have a unique ID for its identification. Similarly, this ID would be the one used to link the sample data to the human clinical data. Thus, it is important to control this process to make sure the linkage is accurate. In addition, the quality dimensions mentioned before should be controlled also for these data.
+
+  In the collection point module of the CovIT from the Czech Republic, the unique ISIN request number is generated using online communication with ISIN and a unique barcode containing ISIN request number is generated and can be print on the label together with name, surname, personal identification number and date and time of sampling. Correct identification and processing of the sample in the laboratory is ensured by this labelling. The second module of CovIT software is a module for SARS-CoV-2 testing laboratories. After delivery of the sample to the laboratory, it can be registered through barcode scanning as well as unique ISIN request number or personal ID filling. All data are transferred from the collection point module which ensures the data accuracy. All important parameters related to the sample identification, storage (ID and position of the plate) and results are collected in the laboratory module.
```

Comment:
In the collection point module of the CovIT from the Czech Republic, the unique ISIN request number is generated using online communication with ISIN and a unique barcode containing the ISIN request number is generated and can be printed on the label together with name, surname, and personal identification number and the date and time of sampling. This labelling ensures the correct identification and processing of the sample in the laboratory. The second module of CovIT software is a module for SARS-CoV-2 testing laboratories. After delivery of the sample to the laboratory, it can be registered through barcode scanning and a unique ISIN request number or personal ID filling. All data are transferred from the collection point module, which ensures data accuracy. All important parameters related to the sample identification, storage (ID and plate position) and results are collected in the laboratory module.